lib/create-contentful-api.ts

```diff
@@ -512,234 +511,49 @@ export default function createContentfulApi<OptionType extends ChainOptions>(
   }
 
   async function getEntry<Fields extends FieldsType>(id: string, query: EntryQueries = {}) {
-    return makeGetEntry<Fields>(id, query, options) as unknown
+    return await makeGetEntry<Fields>(id, query, options)
```

Comment:
Correct me if I'm wrong, but I think `await` is redundant. The function is async meaning it will wrap the return value in a promise either way, no?

Reply:
You’re absolutely right. My IDE complained about the missing await because of the unknown typecast. Now that the cast is gone we don’t need it anymore.